data-model/SDP-sample-records/textual-artifact_sample1.json

```diff
@@ -0,0 +1,65 @@
+{
+  "id": "ark:/21198/adf20c1f",
+  "type": "Textual Artifact",
+  "label": "Sinai Syriac 2A, ff. 1-152v, Gospels",
+  "unit_start": {
+    "parent_id": "ark:/21198/z1x07bdf",
+    "locus": "1"
+  },
+  "unit_end": {
+    "parent_id": "ark:/21198/z1x07bdf",
+    "locus": "152"
+  },
+  "work": "https://sinaimanuscripts/library.ucla.edu/work/1",
+  "associated_persons": [
+    {
+      "id": "https://sinaimanuscripts/library.ucla.edu/person/42",
+      "attested_name_NS": "ܕܩܕܝܫܐ ܡܪܝ ܝܘܚܢܢ",
```

Comment:
One thing I have overlooked relatively consistently is the use of JSON language mappings...names are a good candidate here, and we should probably at least make a ticket/issue about combing back through and updating all the places we need to incorporate language information

Reply:
Need first to determine where it would go and also the rules for using these.

Reply:
Need first to determine where it would go and also the rules for using these.

Reply:
Created a JIRA ticket for this: https://jira.library.ucla.edu/browse/SLDP-629

Reply:
Created a JIRA ticket for this: https://jira.library.ucla.edu/browse/SLDP-629